script.js

```diff
@@ -0,0 +1,7 @@
+import http from "k6/http";
```

Comment:
:) I don't think you meant to include this file.

Reply:
tks